test/functional/buildAndPackage/src/net/adoptopenjdk/test/VendorPropertiesTest.java

```diff
@@ -67,8 +67,8 @@ public VendorPropertiesTest() {
      */
     @Test
     public void javaVersionPrintsVendor() {
-        // Skip test on JDK8 for non-Hotspot JDKs.
-        if (!jdkVersion.isNewerOrEqual(9) && !jdkVersion.usesVM(VM.HOTSPOT)) {
+        // Skip test on JDK8 with OpenJ9 VM, as java -version output will not include a vendor name by design.
+        if (jdkVersion.usesVM(VM.OPENJ9) && !jdkVersion.isNewerOrEqual(9)) {
```

Comment:
Prefer `if (jdkVersion.usesVM(VM.OPENJ9) && jdkVersion.isOlderThan(9)) {`

Reply:
Fixed, and I also tweaked the names of the other methods for consistency. Your thoughts?